robot_calibration/CMakeLists.txt

```diff
@@ -147,13 +147,13 @@ target_link_libraries(simple_joint_state_listener
   ${CERES_LIBRARIES}
 )
 
-target_link_libraries(calibrator
-  ${catkin_LIBRARIES}
-  # ${PROJECT_NAME}
-  -luuid
-  yaml-cpp
-  ${CERES_LIBRARIES}
-)
+# target_link_libraries(calibrator
+#   ${catkin_LIBRARIES}
+#   # ${PROJECT_NAME}
+#   -luuid
+#   yaml-cpp
+#   ${CERES_LIBRARIES}
+# )
```

Comment:
This really shouldn't be in this PR, if we're being honest, but it's ok as it's commented out.